File: CS/13_Conversion/ToDoc.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using Spire.Pdf;

namespace ToDoc
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            // Load a PDF document from the specified file path.
            String file = @"..\..\..\..\..\..\Data\ToDoc.pdf";
            PdfDocument doc = new PdfDocument();
            doc.LoadFromFile(file);

            // Convert the loaded PDF document to a DOC file.
            doc.SaveToFile("ToDoc.doc", FileFormat.DOC);

            // Close the PDF document.
            doc.Close();

            //Launch the file.
            System.Diagnostics.Process.Start("ToDoc.doc");
        }
    }
}

File: VB/13_Conversion/ToDoc.vb
Imports Spire.Pdf

Namespace ToDoc
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles button1.Click
			' Specify the file path of the PDF document to be loaded
			Dim file As String = "..\..\..\..\..\..\Data\ToDoc.pdf"

			' Create a new PdfDocument object
			Dim doc As New PdfDocument()

			' Load the PDF document from the specified file path
			doc.LoadFromFile(file)

			' Convert the PDF document to a DOC file format
			doc.SaveToFile("ToDoc.doc", FileFormat.DOC)

			' Close the PDF document
			doc.Close()

			' Launch the file
			Process.Start("ToDoc.doc")
		End Sub
	End Class
End Namespace
